pkg/lib/authn/authenticator/service/service.go

```diff
@@ -311,12 +312,13 @@ func (s *Service) WithSpec(ai *authenticator.Info, spec *authenticator.Spec) (bo
 		changed = (newAuth != a)
 		return changed, newAuth.ToInfo(), nil
 	case model.AuthenticatorTypeOOBEmail:
```

Comment:
Case switch does not fall through implicitly in Golang. Also, if you have tested update email locally, this will result in a crash.